front/lib/api/assistant/conversation.ts

```diff
@@ -638,15 +649,117 @@ export async function* editUserMessage(
     message: UserMessageType;
     content: string;
   }
-): AsyncGenerator<UserMessageNewEvent | AgentErrorEvent> {
+): AsyncGenerator<UserMessageNewEvent | UserMessageErrorEvent> {
```

Comment:
I'm not sure I see how this type checks you should be yielding stuff not returning stuff?

Reply:
All you errors are not yield here?

Reply:
Actually, the point is moot in the code with single transaction so this can be ignored